### ISSUE TITLE
GUI "ignore" option for runs

### ISSUE THREAD

**Issue by Lewington-pitsos** [tensorflow/tensorboard#3702]
If I'm reading https://github.com/tensorflow/tensorboard/issues/785 correctly the main objection to having a delete function is to avoid write access. This seems sensible, but I can't see anything wrong with users asking tensorboard to "ignore" certain runs (as suggested in  https://github.com/tensorflow/tensorboard/issues/785) and have them avoid showing up in the main "runs" list. 

Keeping a "spam folder" of unwanted runs could also help the user make manual deletions. E.g. users can ask tensorboard for all the ids of runs in the spam folder and then delete them all manually in a single cli command. 

Deleting the files manually is quite cumbersome in my experience and involves a lot of switching back and fourth to make sure you're targeting the right ids. It's also something I find myself wanting to do a lot. As a somewhat inexperienced practitioner I'll often just abandon a whole log directory because it's too cluttered up by failed runs.  

Happy to look into this myself if I get the green light from someone.

**Comment by wchargin** [comment]
Hi @Lewington-pitsos (and sorry for the delay)! This sounds reasonable
to me. I do think that we would want to think through some of the
details: e.g., if tags only exist for hidden runs, do we hide the tags?
what if an entire dashboard only has data for hidden runs? is the
hidden-run state persisted in the fragment? and so on.

Realistically, I don’t think it’s likely that we’ll work on this in the
short term; @stephanwlee, do you think that this is something that we’d
like to accept contributions for, given your context on ongoing UI
changes?

**Comment by Lewington-pitsos** [comment]
Hey @wchargin, I definitely agree that a change like this would have to be well planned out before any work started. If it turns out that the UI is currently in flux, perhaps we could try something a bit simpler for now like having a button that lets you copy all the names of un-selected files to the clipboard for an easy mass `rm -r`.  

I know personally I'd be using a feature like that all the time.

**Comment by Lewington-pitsos** [comment]
closing in favor of #3735
